### ISSUE TITLE
ch06/overfit_weight_decay.pyのweight_decay_lambdaの切り替えについて

### ISSUE THREAD

**Issue by hnakamur** [oreilly-japan/deep-learning-from-scratch#11]
「6.4.1 過学習」と「6.4.2 Weight decay」では共に ch06/overfit_weight_decay.py を実験用のコードとして使用しています。が、以下のコードの部分を切り替える必要がありました。

ch06/overfit_weight_decay.py の以下の部分は 「6.4.2 Weight decay」用のコードになっています。

```
#weight_decay_lambda = 0
weight_decay_lambda = 0.1
```

「6.4.1 過学習」用に使うときは上の箇所を

```
weight_decay_lambda = 0
#weight_decay_lambda = 0.1
```

とする必要がありました。本文中のコードと ch06/overfit_weight_decay.py を見比べれば編集が必要なことがわかりましたが、最初気づかずに ch06/overfit_weight_decay.py を実行してグラフの形が実際に実行した結果と合わずに悩みました。

「6.4.1 過学習」の「（該当ファイルはch06/overfit_weight_decay.py）」のところに上記のような編集が必要な旨を追記するか、あるいはファイルを分けたほうが親切かと思います。

**Comment by koki0702** [comment]
ご指摘、ありがとうございます。

> 「6.4.1 過学習」の「（該当ファイルはch06/overfit_weight_decay.py）」のところに上記のような編集が必要な旨を追記するか、あるいはファイルを分けたほうが親切かと思います。

確かにその通りですね。ソースコードにその旨追記しました。
ありがとうございます。
